cfgov/research_reports/models.py

```diff
@@ -47,6 +51,10 @@ def get_researchers():
     ])
 
 
+def _get_deploy_environment():
+    return getattr(settings, 'DEPLOY_ENVIRONMENT', None)
```

Comment:
This isn't set my default in the local docker setup. This could be set or the `getattr` call could default to `'local'` instead of `None`